ZuluContent/Zulu/Engines/Races/ZuluRace.cs

```diff
@@ -0,0 +1,243 @@
+using System;
+using System.Collections.Generic;
+using Server;
+using Server.Engines.Craft;
+using Server.Engines.Magic;
+using Server.Items;
+using Server.Mobiles;
+using Server.Spells;
+using ZuluContent.Zulu.Engines.Magic.Hooks;
+
+namespace Scripts.Zulu.Engines.Races
+{
+    [PropertyObject]
+    public class ZuluRace : IEnchantmentHook
```

Comment:
Looks good for now :+1: need to have a think for Phase 2 on how to make this abstract enough that people can add their own races via config files.